tools/itsg33-to-gh-issues/script.py

```diff
@@ -1,31 +1,229 @@
+"""
+The purpose of this script is to take the CSV file with the ITSG-33 controls and create issues in github for each control.
+The script will create the issue title, body, and labels. The body will have the control definition, class, supplemental guidance,
+ references, general guide, suggested placeholder values, profile specific notes, suggested assignment, and support teams.
+"""
+
 import csv
 import requests
 import json
 import os
+import string
 
+"""
+env vars:
+REPO = owner/repo
+GITHUB_TOKEN = github token to create issues
+CSV_FILE = path to csv file
+DEBUG = True/False to print debug info
+"""
 REPO = os.getenv("REPO")
 GITHUB_TOKEN = os.getenv("GITHUB_TOKEN")
+CSV_FILE = "controls.csv"
+DEBUG = True
+FAMILY = 0
+CONTROL_ID = 1
+ENHANCEMENT = 2
+CONTROL_NAME = 3
+CONTROL_CLASS = 4
+CONTROL_DEFINITION = 5
+REFERENCES = 7
+SUPPLEMENTAL_GUIDANCE = 6
+IT_SECURITY_FUNCTION = 8
+IT_OPERATION_GROUP = 9
+IT_PROJECTS = 10
+PHYSICAL_SECURITY_GROUP = 11
+PERSONNEL_SECURITY_GROUP = 12
+LEARNING_CENTER = 13
+GENERAL_GUIDE = 14
+SUGGESTED_PRIORITY = 15
+SUGGESTED_PLACEHOLDER_VALUES = 16
+PROFILE_SPECIFIC_NOTES = 17
```

Comment:
There's nothing wrong with how you've done this, but I often see long list of related values like this defined as an enum:
https://www.geeksforgeeks.org/enum-in-python/

I find it helps make it explicit that all these values are part of the same group.

Reply:
Done! Thanks for the tip!